packages/discovery-provider/plugins/pedalboard/apps/solana-relay/src/routes/launchpad/constants.ts

```diff
@@ -0,0 +1,7 @@
+export const SOL_MINT = 'So11111111111111111111111111111111111111112'
```

Comment:
this can be imported as import { NATIVE_MINT } from '@solana/spl-token'

just FYI, idrc